File: solutions/uri/1009/1009.cs
using System;

class Solution {
    static void Main() {
        double b, c;

        _ = Console.ReadLine();
        b = Convert.ToDouble(Console.ReadLine());
        c = Convert.ToDouble(Console.ReadLine());

        Console.WriteLine("TOTAL = R$ {0:F2}", b + c * 0.15);
    }
}

File: solutions/uri/1009/1009.scala
object Main {
    def main(args: Array[String]) {
        val _ = io.StdIn.readLine()
        val b = io.StdIn.readLine().toDouble
        val c = io.StdIn.readLine().toDouble

        println("TOTAL = R$ %.2f".format(b + (c * 0.15)))
    }
}
